roles/openshift_metrics/tasks/cleanup.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- name: remove metrics components
+  command: >
+    {{ openshift.common.client_binary }} -n '{{ project }}'
```

Comment:
we probably want to copy out a kubeconfig and provide that with `--config` for our commands that use the common binary to make sure we have the proper access. There was discussion that an 'oc logout' could potentially break our Ansible roles and that should help us get around that. We copy one into our working temp directory in logging.